vispy/scene/cameras/base_camera.py

```diff
@@ -108,6 +109,19 @@ def depth_value(self, value):
             raise ValueError('depth value must be positive')
         self._depth_value = value
 
+    @property
```

Comment:
Actually, I'm not sure translation speed is the best way to do this. When you pan a view, the place you clicked on should always remain under the mouse cursor. For orthographic projections this is easy, but for perspective projections you need to decide the distance to the plane that will remain stationary under the cursor. The perspective cameras already have a notion of a center-of-view point, so we should already be able to define that distance..?